tests/nnstreamer_plugins/unittest_plugins.cpp

```diff
@@ -153,7 +154,8 @@ TEST (test_tensor_transform, properties)
 #ifdef HAVE_ORC
   g_object_set (transform, "acceleration", !default_accl, NULL);
   g_object_get (transform, "acceleration", &accl, NULL);
-  EXPECT_EQ (!default_accl, accl);
+  /** expect TRUE, which is !default_accl */
```

Comment:
```expect FALSE``` default accl with orc is TRUE.

Reply:
Do we need to change line 152 also?

Reply:
line 1212 
EXPECT_EQ (res_silent, !silent);

Reply:
@jaeyun-jung 

> expect FALSE default accl with orc is TRUE.
Right. I will change the comment.

> Do we need to change line 140 and 152 also? 
> line 1212
EXPECT_EQ (res_silent, !silent);

Only EXPECT_/ASSERT_* (!FALSE, *) cases might occur such warnings.